cli/command/manifest/fetch.go

```diff
@@ -185,16 +188,21 @@ func loadManifestList(transaction string) (foundImages []fetcher.ImgManifestInsp
 	return foundImages, nil
 }
 
-func storeManifest(imgInspect fetcher.ImgManifestInspect, name, transaction string) error {
-	// Store this image manifest so that it can be annotated.
-	// Store the manifests in a user's home to prevent conflict.
+// TODO: some of this should be abstracted out to a struct responsible for
+// managing the local manifests
+func storeManifest(image fetcher.ImgManifestInspect, namedRef reference.Named, transactionID string) error {
+	if transactionID == "" {
```

Comment:
we should never end up in the situation where we're storing a manifest that isn't part of a manifest list transaction (unless we add the ability to pull down a single manifst and store it for some reason). so, this should actually just return an error in this case.